README.md

```diff
@@ -1,70 +1,34 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> Math Magicians
```

Comment:
- Please kindly some description about your project to give users an overview of what the project is all about